### PR TITLE
Update institution-landing-en.html

### PR DESCRIPTION
**Pull request by jmealing**
- Remove provisional classes
- Add Minister's photos
- Contact us on it's own in the band
- Change doormat column count to have same behaviour as homepage
- Margin class adjustment to fix Social Media component issues (hopefully)

To do:
* [x] Need a space to preview to validate it looks and behaves correctly
* [ ] FR version
* [x] Add CSS for the bold class so About the Institution remain bolded when not in the well (well-bold class no longer affecting them)
* [x] DTO to create documentation

### CODE REVIEW

**Comment by duboisp** [comment]
For your review @duboisp later this week

**Review by duboisp — CHANGES_REQUESTED** [review]
@Garneauma do you think we can have a little bit of JS that can detect this is an ILP page  based on the number/type of component on the page? That way we could be able to implement a fall-back to add the CSS class `page-type-ilp` on the page body element.

We will need to mention it in the instruction, for MWS where the author must choose the content type "navigation page - institutional profile" via the basic page property but I don't think they would necessary think of it proactively. With this method, we can also throw a warning about that in the browser console.

Note that we will need to update the page context feature in the MWS to implement the bodyClass `page-type-ilp` when the content type "navigation page - institutional profile" is choosen.

What do you think? Open to suggestion/solution.

cc @GormFrank @jonasg-gc

**Comment by Garneauma** [comment]
Pre-approved upon successful review, conflict resolution, added logic to detect the template.

File: templates/institutional-landing/institutional-landing-fr.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+		</div>
+	</section>
```
Comment (duboisp) [inline templates/institutional-landing/institutional-landing-fr.html:63]: Note: to add a new sample in gc-srvinfo for this 3 column configuration

https://github.com/wet-boew/GCWeb/tree/master/components/gc-servinfo/samples

File: templates/institutional-landing/institutional-landing-fr.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+title: "[Nom de l'institution]"
+layout: no-container
+language: "fr"
+altLangPage: "institutional-landing-en.html"
+overwriteBreadcrumbs: true
+dateModified: "2024-03-22"
```
Comment (duboisp) [inline templates/institutional-landing/institutional-landing-fr.html:7]: Put today date

```suggestion
dateModified: "2024-04-15"
```

File: templates/institutional-landing/institutional-landing-en.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
+---
+title: "[Institution name]"
+layout: no-container
+language: "en"
+altLangPage: "institutional-landing-fr.html"
+overwriteBreadcrumbs: true
+dateModified: "2024-03-22"
```
Comment (duboisp) [inline templates/institutional-landing/institutional-landing-en.html:7]: Put today date

```suggestion
dateModified: "2024-04-15"
```

File: templates/institutional-landing/index.json-ld [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+	},
+	"title": {
+		"en": "Institutional landing page",
+		"fr": "Page d'accueil institutionnelle"
+	},
+	"description": {
+		"en": "Institutional landing page template documentation and working example.",
+		"fr": "Documentation et exemple pratique de la page d'accueil institutionnelle"
+	},
+	"modified": "2024-03-26",
```
Comment (duboisp) [inline templates/institutional-landing/index.json-ld:17]: Put today dates

```suggestion
	"modified": "2024-04-15",
```

File: templates/institutional-landing/institutional-landing-fr.html [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+					</div>
+					<div class="col-sm-6">
+						<div class="well well-sm brdr-rds-0 eqht-trgt">
+							<img class="img-responsive full-width" src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" />
+							<h3 class="h5"><a class="stretched-link" href="#">[Lien à l'élément en vedette]</a></h3>
+							<p>Brève description de l'élément en vedette.</p>
+						</div>
+					</div>
+				</div>
+			</section>
```
Comment (duboisp) [inline templates/institutional-landing/institutional-landing-fr.html:117]: Use the code sample to be included instead

```suggestion
		{%- include components/gc-features/features.html -%}
```

File: templates/institutional-landing/index.json-ld [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers/publishers adding the component manually.",
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement."
+			},
+			"instructions": {
+				"en": [
+					"Refer to the working example for a code sample.",
+					"Refer to the official guidance for more information on how to implement this page template."
+				],
```
Comment (duboisp) [inline templates/institutional-landing/index.json-ld:135]: Add the following similar instruction:
* The author need to add the `page-type-ilp` CSS class on their page `<body>` element. Note: If you don't, the heading level 2 are not going to be at the right size according to the guidance. We did implemented a temporary workaround to give the time to Web CMS to implement it. The workaround are going to be removed at the same time of the demote/deprecated ILP version 1

Also, a little addition for the last instruction item:
```suggestion
				"en": [
					"Refer to the working example for a code sample.",
					"Refer to the working example and the guidance for more information on how to implement this page template."
				],
```

File: templates/institutional-landing/index.json-ld [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+				"fr": [
+					"Reportez-vous à l'exemple pratique pour un exemple de code.",
+					"Référez-vous aux directives officielles pour plus d'informations sur l'implémentation ce modèle de page."
```
Comment (duboisp) [inline templates/institutional-landing/index.json-ld:138]:
```suggestion
					"Référez-vous à l'exemple pratique et aux directives pour plus d'informations sur l'implémentation ce modèle de page."
```

**Review by duboisp — CHANGES_REQUESTED** [review]
There is a few editorial changes, but other than that it looks very good and we will be able to merge after.

File: templates/institutional-landing/_screen-sm-max.scss [hunk 8]
```diff
@@ -0,0 +1,14 @@
+/*
+* Institutional landing page template
+*/
+
+.page-type-ilp {
+	.gc-followus {
```
[the remark this answers is not in the document]
Comment (Garneauma) [reply templates/institutional-landing/_screen-sm-max.scss:6]: Je viens de réaliser que ce changement est propre au gabarit ILP. Il ne sera pas appliqué au gc-follow-us.

File: templates/institutional-landing/institutional-landing-en.html [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+				<li><a href="#">[Top contact task 2]</a></li>
+				<li><a href="#">[Top contact task 3]</a></li>
+			</ul>
+		</section>
+	</div>
+</div>
+
+<div class="container">
+	<div class="row">
+		<div class="col-md-8">{%- include /design-patterns/gc-institution/gc-institution.html -%}</div>
```
[the remark this answers is not in the document]
Comment (Garneauma) [reply templates/institutional-landing/institutional-landing-en.html:36]: The normalization will need to be done at a later date as changing the include to a sample will require changes to more than 20 files.

File: templates/institutional-landing/_screen-sm-max.scss (hunk 8 above)
[the remark this answers is not in the document]
Comment (duboisp) [reply templates/institutional-landing/_screen-sm-max.scss:6]: Parfait, merci de confirmer

File: templates/institutional-landing/institutional-landing-bad.html [hunk 10]
```diff
@@ -0,0 +1,72 @@
+---
+title: "[Institution name]"
+layout: no-container
+language: "en"
+altLangPage: "institutional-landing-fr.html"
```
Comment (duboisp) [inline templates/institutional-landing/institutional-landing-bad.html:5]: Considering there is no translation for this page and it is only used for testing. Just don't provide an `altLangPage` which will auto remove the language toggle link.

```suggestion
```

File: templates/institutional-landing/institutional-landing.js [hunk 11]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @title Institutional landing page template detection
+ * @overview Logic to detect pages that seem to be institutional landing pages without the appropriate class
```
Comment (duboisp) [inline templates/institutional-landing/institutional-landing.js:3]: Can you mention this is temporary measure until that proper fix is implemented by content management system. This code are going to be deprecated and removed at the same time of the version 1 of the ILP template. It just need to be aligned with our technical implementation note.

**Review by duboisp — CHANGES_REQUESTED** [review]
2 little change,
* Remove the alternate page link for the test example
* Add a comment about the limited life span for the JS code

Optional, can done at a later date: The documentation are not showing link to the accessibility assessment we did. Probably the documentation template need to be adjusted.

**Review by duboisp — APPROVED** [review]
Review and tested locally,

Everything work as expected.

**Pull request merged**